### ISSUE TITLE
Improve how we show voting power 

### ISSUE THREAD

**Issue by m-i-k-e** [IntersectMBO/govtool#44]
From a Slack thread:

 

I agree with @Thomas Upfield  that the way we display Voting Power is confusing. This is tied to the decision when the CIP was written to make everyone dReps, regardless of if they wanted to be one or not.What we have is ADA balance in the first place, which is initially ‘potential’ voting power. However, it’s not converted to voting power until it’s delegated. When it’s delegated, what happens? That’s when it get confusing.We really should get our heads together and fix this. This will require, in my view two things: Re-jigging the Voting registration so that ‘non-dRep’ votes (or sole voters if you will) have a place in the system. This fixes one issue: Their voting power will always be their ADA balance.Then we need to be able to show for people who are delegating only their voting power. This will always be their ADA balance, but we need to show who’s it being delegated to. Currently the interface does that, but it could be presented more clearly.Finally, the big issue is when you’re a dRep. There are three wrinkles to this:

You have your own inherent Voting Power based on how much ADA you have in your wallet

This can be delegated to

yourself,

another dRep

undelegated

You have also the possibility of having other people delegate to you. This increases your voting power as a dRep, but does not affect your personal Voting Power based on ADA balance. This is where it gets confusing.

Basically, we’ve not built in a way to display (for dReps) a way where they can see the VP delegated to them, whilst still seeing their own personal ADA balance, which again, may or not be converted to VP.

 

One thing that contributes to this confusion is that the wallet connect button turns into Voting Power as soon as the connections is made. This, now that I think of it, is not correct, if you haven’t delegated your ADA yet. Initially, it’s just ADA, and it gets converted to Voting Power when you delegate. This is a subtle difference, but an important one.
